src/helper_functions.py

```diff
@@ -0,0 +1,62 @@
+from item import Item
+
+"""
+handle_direction takes in the string entered by user and the current room instance from the player,
+creates an attribute variable that concatenates the input string + '_to'
+uses hasattr function to check if the current_room Room instance has the attribute we just defined
+then if it does return the getattr function to get the value of that attribute from that Room instance
+"""
+
+def handle_direction(input, current_room):
+    attribute = input + '_to'
+    if hasattr(current_room, attribute):
+        return getattr(current_room, attribute)
+    else:
+# Print an error message if the movement isn't allowed.
+        print("""
+        
+        **You can't go that way!**
+        Either the map doesn't go that direction or you typed an invalid direction key.
+        Direction keys are: n, s, e, w 
+        Other valid commands: get [item], take [item], drop [item]
+        
+        """)
+        return current_room
```

Comment:
For instance this would make a lot more sense being a player method. So then you would do something like player.move('w')

Reply:
Also it's a bit hard to follow.